src/v0/destinations/wootric/transform.js

```diff
@@ -45,7 +51,7 @@ const identifyResponseBuilder = async (message, destination) => {
 
   const accessToken = await getAccessToken(destination);
   if (!accessToken) {
-    throw new CustomError("Access token is missing", 400);
+    throw new NetworkInstrumentationError("Access token is missing");
   }
```

Comment:
Based on the `getAccessToken` implementation, this validation may not be required. Double-check that, please.
Also, even if we were to throw any error here, it is definitely not the instrumentation error.